tests/rpc.rs

```diff
@@ -0,0 +1,121 @@
+use monero_harness_rs::Client;
+use spectral::prelude::*;
+use std::time::Duration;
+use testcontainers::clients;
+use tokio::time;
+
+#[tokio::test]
+async fn connect_to_monerod() {
+    let docker = clients::Cli::default();
+    let cli = Client::new_with_random_local_ports(&docker);
```

Comment:
Stale variable name :)

Reply:
I don't understand sorry. Is this referencing your comment above about the name of this type?

Reply:
Yes